### ISSUE TITLE
Duplicate timestamps

### ISSUE THREAD

**Issue by captain-munch** [hirosystems/ordinals-api#36]
We have both `genesis_timestamp` and `timestamp` which seem to have duplicate values. 

What would be super useful, is if the `timestamp` was reflective of the most recent tx for the inscription. This would make contextual sense in relation to `address` which is the current holder, versus the `genesis_address` and may be reflective of the bug in https://github.com/hirosystems/ordinals-api/issues/23

Here is an example object that displays the issue: 

```
{
  "id": "fd682b9fcd6fb1450656dcb9220bab18821f3833707ef7225b827f14b56c5fc9i0",
  "number": 2379,
  "address": "bc1py5ur3un0072232w7pmme2v7qexfmkvjt2eqnjx65vf3wcmuxallqatgmea",
  "genesis_address": "bc1py5ur3un0072232w7pmme2v7qexfmkvjt2eqnjx65vf3wcmuxallqatgmea",
  "genesis_block_height": 775057,
  "genesis_block_hash": "000000000000000000068aac09e93b84477b85464b8a57c02e59b9e725e13a1f",
  "genesis_tx_id": "fd682b9fcd6fb1450656dcb9220bab18821f3833707ef7225b827f14b56c5fc9",
  "genesis_fee": "22429",
  "genesis_timestamp": 1675551312000,
  "tx_id": "fd682b9fcd6fb1450656dcb9220bab18821f3833707ef7225b827f14b56c5fc9",
  "location": "fd682b9fcd6fb1450656dcb9220bab18821f3833707ef7225b827f14b56c5fc9:0:0",
  "output": "fd682b9fcd6fb1450656dcb9220bab18821f3833707ef7225b827f14b56c5fc9:0",
  "value": "10000",
  "offset": "0",
  "sat_ordinal": "132406623762452",
  "sat_rarity": "common",
  "sat_coinbase_height": 26481,
  "mime_type": "image/jpeg",
  "content_type": "image/jpeg",
  "content_length": 88664,
  "timestamp": 1675551312000
}
```

Ideally `timestamp` would be `1678006652000` reflecting [the most recent tx](https://mempool.space/tx/5de3485ca46b8132c67b6bb1b41f2d84a1f85a771206076666db5e237b57b1b8) for this inscription.

**Comment by rafaelcr** [comment]
Hi @muncheruniverse that's correct. This will be addressed in #23

**Comment by rafaelcr** [comment]
Closing as duplicate
